mon-pix/app/controllers/authenticated/user-account/language.js

```diff
@@ -9,12 +10,33 @@ export default class UserAccountPersonalInformationController extends Controller
   @service currentDomain;
   @service locale;
 
+  @tracked showLanguageUpdatedMessage = false;
+  @tracked showLanguageOptions = false;
+
   @action
   async onLanguageChange(language) {
     if (!this.currentDomain.isFranceDomain) {
       await this.currentUser.user.save({ adapterOptions: { lang: language } });
 
       this.locale.setLocale(language);
+      this.displayLanguageUpdatedMessage();
+    }
+  }
+
+  displayLanguageUpdatedMessage() {
+    this.showLanguageUpdatedMessage = true;
+  }
+
+  @action
+  onToggleShowLanguageOptions() {
+    this.showLanguageOptions = !this.showLanguageOptions;
+    if (this.showLanguageOptions) {
+      this.hideLanguageUpdatedMessage();
     }
   }
+
+  @action
+  hideLanguageUpdatedMessage() {
+    this.showLanguageUpdatedMessage = false;
+  }
```

Comment:
Dans le cas où le décorateur `@action` n'est pas nécessaire, alors je suggères de passer en méthode "privée" et de déplacer la méthode en bas de la classe.

```suggestion
  _hideLanguageUpdatedMessage() {
    this.showLanguageUpdatedMessage = false;
  }
```

Reply:
Pareil cette méthode faisait partie des tests, elle n'est pas utilisée = à supprimer